app/templates/pages/jobs/software-architect.html

```diff
@@ -4,128 +4,99 @@
 {% block job_content %}
 
 <div class="row">
-    <div class="col-lg-12">
-        <p>
-            In order to promote these aims, we are currently seeking a capable and motivated
-            <b>Software Architect</b> to join our agile development teams in our office at the heart of Berlin.
-            You will be responsible for having your eye on the big picture that guides the development
-            of our complex software products. 
-        </p>
-    </div>
+	<div class="col-lg-12">
+		<p>
+			We are looking for a capable and motivated <b>Software Architect</b> to join our agile
+			development teams in our office at the heart of Berlin. You will be responsible for
+			keeping an eye on the big picture and for assisting our developers in creating well
+			designed software and migrating away from legacy solutions.
+		</p>
+	</div>
 </div>
 
 <div class="row">
-    <div class="col-lg-12">
-        <h2>About you</h2>
-
-        <p>As a Software Architect you have/are:</p>
-
-        <ul>
-            <li>Experience in software development and coding in various languages</li>
-	    <li>Excellent knowledge of software and application design and architecture</li>
-	    <li>Eye level communicator with a proactive attitude for solving problems</li>
-	    <li>Able to translate your ideas and decisions in a clear and intelligible way to technical and non-technical folks alike</li>
-	    <li>Team-focused with strong collaborative skills</li>
-	    <li>Enjoying mentoring other developers and helping them understand complex structures</li>
-	    <li>Excellent knowledge of UML and other modeling methods</li>
-	    <li>Understanding of software quality assurance principles</li>
-	    <li>A technical mindset with great attention to detail</li>
-	    <li>High quality organizational and leadership skills</li>
-        </ul>
-    </div>
+	<div class="col-lg-12">
+		<h2>About you</h2>
+
+		<ul>
+			<li>You care about software; you have a passion which you can clearly convey by your actions rather than personal statements on your CV alone</li>
+			<li>You have deep understanding of software quality, design and architecture and of system architecture</li>
+			<li>You can recognize legacy code and know how to effectively deal with it</li>
+			<li>You are able to communicate fluently in English in a multidisciplinary team</li>
```

Comment:
Why this preference? Because "eye level communicator"?

To me this point seems to be about communication and a proactive attitude, which are two different things, hence the separation I did. Do you think they should be together for some reason? If not, I'd leave the proactive bit alone and modify the communication point to include "eye level communicator". In doing so we should not drop the "English" bit though, since that is important :)